### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -62,10 +64,15 @@ def read(*rnames):
...
         'Operating System :: OS Independent',
         'Topic :: Internet :: WWW/HTTP',
-        'Framework :: Zope :: 3'],
+    ],
     zip_safe=False,
-    entry_points={'distutils.commands': 'ftest = zope.testrunner.eggsupport:ftest'}
-    )
+    entry_points={
+        'distutils.commands': 'ftest = zope.testrunner.eggsupport:ftest'
+    }
```
Comment (mgedmin) [inline setup.py]: TBH I think this can be dropped entirely.

File: src/refline/srccheck/pyflakes.py [hunk 2]
```diff
@@ -5,13 +5,14 @@
 """
 Implementation of the command-line I{pyflakes} tool.
 """
-
+from __future__ import print_function
```
Comment (mgedmin) [inline src/refline/srccheck/pyflakes.py:8]: This is probably worth a changelog note.

(How did version 1.0 claim "official Python 3 support" with this kind of SyntaxError lurking it the source??)

**Review by mgedmin — APPROVED** [review]
Lgtm

File: setup.py (hunk 1 above)
Comment (icemac) [reply setup.py]: Done.

File: src/refline/srccheck/pyflakes.py (hunk 2 above)
Comment (icemac) [reply src/refline/srccheck/pyflakes.py:8]: okay.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**